src/BUILD

```diff
@@ -6,9 +6,7 @@ load(":rule_size_test.bzl", "rule_size_test")
 
 exports_files(["jdeps_modules.golden"])
 
-# We avoid using the `head` tool's `-c` option, since it does not exist on
-# OpenBSD; here we use `dd` instead.
-md5_cmd = "set -e -o pipefail && echo $(SRCS) | sort | xargs %s | %s | dd bs=32 count=1 > $@"
+md5_cmd = "set -e -o pipefail && %s $(SRCS) | %s | %s > $@"
```

Comment:
If this were "set -e -o pipefail && %s $(SRCS) | **sort |** %s | %s > $@"
then that would be immune to Bazel reordering $SRCS. (if we still want to preserve that intent)

Reply:
The current implementation should be deterministic per (Bazel version, repo version), which should be good enough?

Reply:
Sure. It was more a suggestion. I'm fine with what you have.
I can merge when I am back at my desk.